terraform/app/modules/ecs_service/variables.tf

```diff
@@ -0,0 +1,79 @@
+variable "environment" {
+  type        = string
+  description = "Application environment (for example production or staging)"
+  nullable    = false
+}
+
+variable "server_type" {
+  type        = string
+  description = "Type of server to be deployed. This is set as an environment variable in the main container, and is used to determine how the application is launched"
```

Comment:
Can we pass the server_type as part of the `task_config` environment?

Reply:
As discussed, we won't do it since the server type is also referenced in resource names